build_library/catalyst_toolchains.sh

```diff
@@ -16,10 +16,10 @@ configure_target_root() {
     local cross_chost=$(get_board_chost "$1")
     local profile=$(get_board_profile "${board}")
 
-    CHOST="${cross_chost}" \
-        ROOT="/build/${board}" \
-        SYSROOT="/usr/${cross_chost}" \
-        _configure_sysroot "${profile}"
+    CBUILD="$(portageq envvar CBUILD)" \
+        CHOST="${cross_chost}" \
+        ROOT="$ROOT" SYSROOT="$ROOT" \
+        _configure_sysroot "${CROSS_PROFILES[${cross_chost}]}"
```

Comment:
Another odd thing in this diff, ROOT needs to point to the target boards root, SYSROOT needs to point to the cross-compiler's sysroot to provide the initial dependencies of the toolchain, they don't yet exist in ROOT. I presume this change was coppied from elsewhere where ROOT and SYSROOT were indeed the same.